lib/rspec/core/profiler.rb

```diff
@@ -0,0 +1,29 @@
+module RSpec
+  module Core
+    # @private
+    class Profiler
```

Comment:
I'm strongly against polluting the formatters namespace with more things that are not formatters. The reporter is not in the formatters namespace ergo this isn't.